frontend/src/components/activity/content/storyboard/StoryboardRowDefault.vue

```diff
@@ -7,8 +7,8 @@
         class="drag-and-drop-handle"
         :disabled="isLastSection"
         :aria-label="$tc('global.button.move')"
-        @keydown.down="$emit('moveDown', itemKey)"
-        @keydown.up="$emit('moveUp', itemKey)"
+        @keydown.down="$emit('move-down', itemKey)"
```

Comment:
@manuelmeister 
Should it be possible to move the storyboard entries here around with the keyboard?
how do i test this?